modules/checkout/src/Plugin/Commerce/CheckoutPane/Login.php

```diff
@@ -243,8 +242,11 @@ public function validatePaneForm(array &$pane_form, FormStateInterface $form_sta
     $values = $form_state->getValue($pane_form['#parents']);
     $username = $values['returning_customer']['name'];
     $password = trim($values['returning_customer']['password']);
+    // Setting url for "forgotten password" link in the error message.
+    $route_parameters = !empty($username) ? array('name' => $username) : array();
```

Comment:
With all other changes lines in this PR are short array syntax, but this line uses long syntax, let's change that as well.